### PR TITLE
https://issues.redhat.com/browse/ACM-5559 add .yaml callout

### PR DESCRIPTION
**Pull request by faaden123**

### CODE REVIEW

**Comment by faaden123** [comment]
I think I may have added a callout where I didn't need to( the 2 in this case). Let me know if this may need to be removed!

File: business_continuity/volsync/volsync_replicate.adoc [hunk 1]
```diff
@@ -133,7 +133,7 @@ Replace `destination-ns` with the namespace where your replication destination i
 
 .. Create a new YAML file called `replication_destination` and copy the following content:
 +
-[source,yaml]
+[]
```
Comment (oafischer) [inline business_continuity/volsync/volsync_replicate.adoc]: You might have accidentally deleted that line, make sure to add that back in.

File: business_continuity/volsync/volsync_replicate.adoc [hunk 2]
```diff
@@ -142,17 +142,16 @@ metadata:
   namespace: <destination-ns>
 spec:
   rsyncTLS:
-    serviceType: LoadBalancer
+    serviceType: LoadBalancer <2>
```
Comment (oafischer) [inline business_continuity/volsync/volsync_replicate.adoc]: I would recommend starting with 1 first, then 2. You can rearrange the notes below to match the order.

File: business_continuity/volsync/volsync_replicate.adoc (hunk 1 above)
Comment (faaden123) [reply business_continuity/volsync/volsync_replicate.adoc]: Just added it back in

File: business_continuity/volsync/volsync_replicate.adoc (hunk 2 above)
Comment (faaden123) [reply business_continuity/volsync/volsync_replicate.adoc]: Just made these changes, let me know if they look good now !

Comment (oafischer) [reply business_continuity/volsync/volsync_replicate.adoc]: I'm not seeing the changes, did you commit them?

**Review by oafischer — APPROVED** [review]
Looks great, good practice on making edits too!

**Pull request merged**